addons/skin.estuary/1080i/DialogContextMenu.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <window>
-	<defaultcontrol>1000</defaultcontrol>
+	<defaultcontrol>996</defaultcontrol>
```

Comment:
no real hard reason, but I think it makes more sense to put focus on the grouplist instead of a button template?